docs/src/examples/FOBP_example2.jl

```diff
@@ -1,90 +0,0 @@
-# # Foundations of Bilevel Programming: Example 2
```

Comment:
You probably didn't mean to delete this file?

Reply:
sorry, I am learning.

Reply:
You need to add it back and push a new commit. The easiest way is to checkout `master`, copy the file, then checkout `example3` and paste.